examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +70,12 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds_mt, freqs_mt = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
+                                   fmin=fmin, fmax=fmax, picks=picks, n_jobs=1)
+ax.plot(freqs_mt, 10 * np.log10(psds_mt).T)
+ax.set(title='Multitaper PSD', xlabel='Frequency',
+       ylabel='Power Spectral Density (dB)')
```

Comment:
I just tried this plot. It seems that `plot_psd` shows the mean and std, not all the channels. But here you show all 4 channels. Might be a bit confusing ... I think they should show the same thing.